Client/src/Pages/Infrastructure/Details/index.jsx

```diff
@@ -101,10 +103,25 @@ BaseBox.propTypes = {
  * @returns {React.ReactElement} Stat box component
  */
 const StatBox = ({ heading, subHeading }) => {
+	const theme = useTheme();
 	return (
-		<BaseBox>
-			<Typography component="h2">{heading}</Typography>
-			<Typography>{subHeading}</Typography>
+		<BaseBox
+			sx={{
+				padding: `${theme.spacing(8)} ${theme.spacing(6)}`,
+			}}
+		>
+			<Typography
+				component="h2"
+				fontSize={TEXT_FONT_SIZE}
+			>
+				{heading}
+			</Typography>
+			<Typography
+				fontSize="16px"
```

Comment:
Hardcoded values for font size are a no go, please reference font sizes from the theme